e2e-tests/resources/authd-msentraid/broker.resource

```diff
@@ -0,0 +1,296 @@
+*** Settings ***
+Documentation       MS Entra ID specific resources for the tests
+
+Resource            kvm.resource
+Resource            ../authd/utils.resource
+Library             ./Browser.py    AS    Browser
+Library             Hid.py    AS    Hid
+
+*** Variables ***
+${ENTRAID_RESOURCES}    ${CURDIR}
+
+${AUTHD_BROKER_CFG}    /etc/authd/brokers.d/msentraid.conf
+${ENTRAID_BROKER_CFG}    /var/snap/authd-msentraid/current/broker.conf
+${ENTRAID_BROKER_CFG_DIR}    /var/snap/authd-msentraid/current/broker.conf.d
+
+
+*** Keywords ***
+Enable Edge Broker
+    Open Terminal In Sudo Mode
+    Run Command In Terminal    snap refresh authd-msentraid --edge
+    Close Terminal In Sudo Mode
+
+
+Disable Broker And Purge Config
+    Open Terminal In Sudo Mode
+    Run Command In Terminal    snap stop authd-msentraid
```

Comment:
This step fails if I increase the OCR confidence threshold because it's expecting `'@ubuntu:~$'` but the string in the root terminal is `root@ubuntu: /home/ubuntu#`.

Reply:
I fixed that